.docker-compose-okteto.yml

```diff
@@ -0,0 +1,25 @@
+version: "3"
+services:
+  mongo:
+    image: "mongo:latest"
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=root
+      - MONGO_INITDB_ROOT_PASSWORD=password
+  growthbook:
+    build: .
+    ports:
+      - "3000:3000"
+      - "3100:3100"
+    depends_on:
+      - mongo
+    environment:
+      - MONGODB_URI=mongodb://root:password@mongo:27017/
+      - APP_ORIGIN=https://growthbook-3000-${OKTETO_NAMESPACE}.cloud.okteto.net
+      - API_HOST=https://growthbook-3100-${OKTETO_NAMESPACE}.cloud.okteto.net
+      - JWT_SECRET=${JWT_SECRET}
+      - IS_CLOUD=true
+      - SSO_CONFIG=${SSO_CONFIG}
```

Comment:
Might want to add `DISABLE_TELEMETRY=debug` here.  We collect anonymous usage data when people use GrowthBook and we don't want these preview environments to pollute that data.  

Adding this will cause it to do `console.log` instead of actually sending the usage data, so we'll still be able to verify that telemetry works.